htdocs/core/modules/import/import_csv.modules.php

```diff
@@ -612,7 +612,7 @@ public function import_insert($arrayrecord, $array_match_file_to_database, $obji
 										break;
 									}
 									$classinstance = new $class($this->db);
-									$res = call_user_func_array(array($classinstance, $method), array(&$arrayrecord));
+									$res = call_user_func_array(array($classinstance, $method), array(&$arrayrecord, $fieldname, &$listfields, &$listvalues, $key - 1));
```

Comment:
Why do you need the &  on $listfields    and   $listvalues  ?
Only  $arrayrecord  (the array that would be used for output should be modified).   $listfield and $listvalues are entry and must remained unchanged during all exposed or you expose the code to bug by having the var modified for one call of method and for another call, input data will not be the same.